### ISSUE TITLE
Docker error "failed to register layer"

### ISSUE THREAD

**Issue by CountCypher** [joplin-vieweb/joplin-vieweb#57]
Hallo,

I'm trying to run joplin-vieweb in a Linux container (Debian 12) on Proxmox. Here is my `docker-compose.yml` file:
```
version: '3.4'

x-common-variables: &common-variables
   ORIGINS: "'https://joplin.mydomain.com'"

services:
  django-joplin-vieweb:
    image: gri38/django-joplin-vieweb:latest
    container_name: joplin-vieweb
    depends_on:
      - joplin-terminal-xapi
    environment:
       <<: *common-variables
    restart: unless-stopped
    ports:
      - 8096:8000
    volumes:
      - joplin:/root/.config/joplin:ro
      - joplin-vieweb:/root/.config/joplin-vieweb
    networks:
      - joplin-net

  joplin-terminal-xapi:
    image: gri38/joplin-terminal-xapi:latest
    container_name: joplin-terminal
    restart: unless-stopped
    volumes:
      - joplin:/root/.config/joplin
    networks:
      - joplin-net

volumes:
  joplin:
  joplin-vieweb:

networks:
  joplin-net: {}
```
After executing `docker-compose up -d`, I receive this error:

```
failed to register layer: failed to Lchown "/app/joplin/lib/node_modules/joplin/node_modules/optionator/CHANGELOG.md" for UID 110779, GID 100 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /app/joplin/lib/node_modules/joplin/node_modules/optionator/CHANGELOG.md: invalid argument
```

I have already followed the hint from the message and changed `/etc/subuid` from this...
```
maddin:100000:65536
```
...to this...
```
maddin:120000:65536
```
...and restarted the Docker service afterwards, but no effect.

Any hints how I can resolve this? 
Thanks in advance. :)

**Comment by gri38** [comment]
Hello.
I think this is an issue about docker rights.
did you try, just in case, to run docker-compose with sudo ?

In my images, I don't run with specific user, so you should run with sudo, or try to handle the permissons...

**Comment by CountCypher** [comment]
Hi @gri38 and thanks for the quick reply. :) Yes, I'm also assuming that it's a docker rights issue. Regarding your suggestion to run docker compose with sudo - that should not be needed, because my user "maddin" is member of the group "docker". Still, I have tried the same, but I get the same error.

I have then tried the same `docker-compose.yml` on my Raspberry Pi 4 and there it works without any problem. So it might be that I receive the error, because I'm trying to run the docker container in an unprivileged Linux container. I will test further and report back here...
